tests/performance/api-pod.yaml

```diff
@@ -30,7 +30,7 @@ spec:
               value: REPLACE_PASSWORD
             - name: KUBECONFIG
               value: /tmp/.kube/config
-          image: quay.io/openshift/origin-cli
+          image: quay.io/sskoryk/performance
```

Comment:
would you consider not a personal organization, but https://quay.io/organization/crw_pr ?

Reply:
Ok

Reply:
If quay.io/sskoryk/performance is built using tests/performance/Dockerfile, then it is a useful information for readmy.md file

Reply:
Ok, I will add it.